### ISSUE TITLE
How does it compare with your Fullpage plugin?

### ISSUE THREAD

**Issue by MakaryGo** [OleVik/grav-plugin-presentation#7]
Hi! Looks really impressive, like all your projects ;)
I only wonder if it's intended as a replacement for [Fullpage plugin](https://github.com/OleVik/grav-plugin-fullpage)?
Could you give some feature comparison perhaps?
Cheers

**Comment by OleVik** [comment]
It is, largely. Fullpage, both the plugin and the underlying Fullpage.js-library, is somewhat simpler in nature compared to the Reveal.js-library the Presentation-plugin uses. The treatment of Pages, FrontMatter and shortcodes is quite similar between the plugins though, thus the core API is still alike. Both plugins facilitate slideshows apt for presenting or displaying in a fullpage-manner.

The Presentation-plugin goes further in some areas: The API is more advanced and easier to extend, and has features like dynamic textsizing. It uses some advanced features for presenter-mode - with optional notes - which allows you to control presentations remotely or locally, and it enhances the editing-experience in Admin.

In summary I think it is fair to say that Fullpage enables navigable, fullpage, Pages, whilst Presentation goes a step further by letting you present, control, and expand this. I've kept Fullpage alive because some might prefer the fullpage.js-library over the reveal.js-library, but development is focused on this one.

Does that clarify it a bit?

**Comment by MakaryGo** [comment]
Absolutely, thanks!

Sent from Mail for Windows 10
